bin/check-codeowners

```diff
@@ -440,30 +446,20 @@ module CheckCodeowners
       end
     end
 
-    def with_splitter_script
-      Tempfile.open do |script|
-        script.puts <<~SCRIPT
-          #!/bin/bash
-          umask 077
-          exec git ls-files -z --cached --ignored --exclude "${1:#{PREFIX_LENGTH + 1}}" \
-            > #{Shellwords.escape(@output_dir)}/${1:0:#{PREFIX_LENGTH}}
-        SCRIPT
-        script.chmod 0o700
-        script.close
-
-        @splitter = script.path
-        yield
-      end
-    end
-
     def run_xargs
```

Comment:
could use a better method name
and a comment on what it's goal is / how it archives that